frontend/heart-code/src/enums/Gender.ts

```diff
@@ -4,7 +4,7 @@ enum Gender {
   NON_BINARY = "non-binary",
   BINARY = "binary",
   OTHER = "other",
-  PREFER_NOT_TO_SAY = "prefer_not_to_say",
+  PREFER_NOT_TO_SAY = "prefer not to say",
```

Comment:
dit is mooi voor de user, maar de API verwacht het wel in de vorm 'prefer_not_to_say', nu zal elke poging tot het kiezen van deze optie leiden tot een 400 error

Reply:
ooh, dat heb ik niet zien aankomen... jammer. maar dan haal ik hier gewoon uit indd, misschien iets voor later dan

Reply:
weer terug gezet